starters/next-drupal-starter/pages/examples/ssg-isr.jsx

```diff
@@ -0,0 +1,101 @@
+import { NextSeo } from "next-seo";
+import { isMultiLanguage } from "../../lib/isMultiLanguage";
+import {
+  getCurrentLocaleStore,
+  globalDrupalStateStores,
+} from "../../lib/drupalStateContext";
+
+import { ArticleGridItem, withGrid } from "../../components/grid";
+import Layout from "../../components/layout";
+
+export default function HomepageTemplate({
+  articles,
+  footerMenu,
+  hrefLang,
+  multiLanguage,
+}) {
+  const HomepageHeader = () => (
+    <div className="prose sm:prose-xl mt-20 flex flex-col mx-auto max-w-fit">
+      <h1 className="prose text-4xl text-center h-full">
+        Articles
+      </h1>
+    </div>
+  );
+
+  const ArticleGrid = withGrid(ArticleGridItem);
+
+  return (
+    <Layout footerMenu={footerMenu}>
+      <NextSeo
+        title="Decoupled Next Drupal Demo"
+        description="Generated by create next app."
+        languageAlternates={hrefLang || false}
+      />
+      <>
+        <HomepageHeader />
+        <div className="mt-8 prose lg:prose-xl max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
+          <p>
+            <em>By default, this starter kit is optimized for SSR and Edge
+              Caching on Pantheon. This example instead uses Incremental Static
+              Regeneration and is provided as a reference for cases where Next.js
+              static generation options would be beneficial.</em>
+          </p>
+        </div>
+        <section>
+          <ArticleGrid
+            data={articles}
+            contentType="articles"
+            multiLanguage={multiLanguage}
+          />
+        </section>
+      </>
+    </Layout>
+  );
+}
+
+export async function getStaticProps(context) {
+  const origin = process.env.NEXT_PUBLIC_FRONTEND_URL;
+  const { locales } = context;
+  // if there is more than one language in context.locales,
+  // assume multilanguage is enabled.
+  const multiLanguage = isMultiLanguage(locales);
+  const hrefLang = locales.map((locale) => {
+    return {
+      hrefLang: locale,
+      href: origin + "/" + locale,
+    };
+  });
+
+  try {
+    const store = getCurrentLocaleStore(
+      context.locale,
+      globalDrupalStateStores
+    );
+
+    const articles = await store.getObject({
+      objectName: "node--article",
+      params: "include=field_media_image.field_media_image",
+    });
+
+    if (!articles) {
+      throw new Error(
+        "No articles returned. Make sure the objectName and params are valid!"
+      );
+    }
+
+    const footerMenu = await store.getObject({
+      objectName: "menu_items--main",
+    });
+
+    return {
+      props: { articles, hrefLang, multiLanguage, footerMenu },
+      revalidate: 60,
+    };
+  } catch (error) {
+    console.error("Unable to fetch data for articles page: ", error);
+    return {
+      notFound: true,
+      revalidate: 5,
+    };
+  }
+}
```

Comment:
This (no new line) makes me wonder if our linting is working 🤔

Reply:
I think that one is on me - I was running this in the dev sandbox because it is way easier for me to make a bunch of crazy edits to content and then throw it all away, but it doesn't look like linting is running correctly due to the lando setup.